e2e-tests/IIoTPlatform-E2E-Tests/Orchestrated/PublishSingleNodesTestTheory.cs

```diff
@@ -289,5 +289,16 @@ public class PublishSingleNodesTestTheory
                 _output.WriteLine($"ErrorMessage: {response.ErrorMessage}");
             }
         }
+
+        [Fact, PriorityOrder(12)]
+        public async void Test_VerfiyNoDataIncomingAtIoTHub() {
+            await Task.Delay(90 * 1000); //wait till the publishing has stopped
+            //use test event processor to verify data send to IoT Hub
+            await TestHelper.StartMonitoringIncomingMessages(_context, 1, 1000, 10000);
```

Comment:
Should the call not look like TestHelper.StartMonitoringIncomingMessages(_context, 0,0,0); ? We don't expect anything...

Reply:
Done.